config.py

```diff
@@ -6,3 +6,4 @@
 sheet_name= 'object'
 expected_irctc_page_title="FlightTicketBooking Next Generation eTicketing System"
 expected_flight_page_title="Air Ticket Booking | Book Flight Tickets | Cheap Air Fare - FlightTicketBooking Air"
+OBJECT_JSON=r"C:\Users\7000026201\PycharmProjects\FlightTicketBooking\Data\object_data.json"
```

Comment:
I cannot run test case on your system always to get the data from the specified path right ??

Reply:
removed absolute path and used relative path